packages/agent/src/agent.ts

```diff
@@ -9,19 +25,40 @@ import {
   KeyDidResolver,
   LogLevel,
   WebDidResolver,
+  CredentialsModule,
+  V2CredentialProtocol,
+  ProofsModule,
+  V2ProofProtocol,
+  AutoAcceptProof,
+  AutoAcceptCredential,
+  MediationRecipientModule,
+  HttpOutboundTransport,
+  WsOutboundTransport,
+  ConnectionsModule,
 } from '@aries-framework/core'
+import {
+  IndyVdrAnonCredsRegistry,
+  IndyVdrIndyDidResolver,
+  IndyVdrModule,
+  IndyVdrSovDidResolver,
+} from '@aries-framework/indy-vdr'
 import { useAgent as useAgentLib } from '@aries-framework/react-hooks'
 import { agentDependencies } from '@aries-framework/react-native'
+import { EnglishMnemonic as _ } from '@cosmjs/crypto'
+import { anoncreds } from '@hyperledger/anoncreds-react-native'
 import { ariesAskar } from '@hyperledger/aries-askar-react-native'
+import { indyVdr } from '@hyperledger/indy-vdr-react-native'
 import { OpenId4VcClientModule } from '@internal/openid4vc-client'
 
+import { indyNetworks } from './indyNetworks'
+
 export const initializeAgent = async (walletKey: string) => {
   const agent = new Agent({
     dependencies: agentDependencies,
     config: {
       label: 'Paradym Wallet',
       walletConfig: {
-        id: 'paradym-wallet-secure',
+        id: 'paradym-wallet-secure-' + `${Math.random()}`,
```

Comment:
This will create a new wallet every time you start the app right is this something we want or was this for testing?

Reply:
Right... Fixed